### PR TITLE
Ot37 list users

### PR DESCRIPTION
**Pull request by noisyBrain**
### In controllers/organization.js:
- New method to list all users

### In dao/user.js
- New class (UserDao)
- New method to get all users from db

### In routes/organization.js
- New route to list all users

### CODE REVIEW

File: controllers/organization.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-const { Organization } = require('../models/');
+const { Organization, User } = require('../models/');
```
Comment (noisyBrain) [inline controllers/organization.js]: Dejé la importación de User sin querer.
Para refactor

File: controllers/organization.js [hunk 2]
```diff
@@ -48,6 +49,20 @@ class OrganizationController {
       });
     }
   }
+
+  static async getUsersList(req, res) {
```
Comment (maxisosa89) [inline controllers/organization.js]: Este método corresponde al controller de users.

File: routes/organization.js [hunk 3]
```diff
@@ -16,4 +17,6 @@ router.post(
   OrganizationController.updateOrganization
 );
 
+router.get('/users', RoleMiddleware.isAdminRole ,OrganizationController.getUsersList)
```
Comment (maxisosa89) [inline routes/organization.js]: Esta ruta corresponde al controller de Users

File: controllers/user-controller.js [hunk 4]
```diff
@@ -55,15 +61,29 @@ class UserController {
         where: { email },
         attributes: ['firstName', 'lastName', 'email', 'image'],
       });
-      console.log(user)
+      console.log(user);
```
Comment (maxisosa89) [inline controllers/user-controller.js]: Quitar console.log

**Review by maxisosa89 — APPROVED** [review]

File: controllers/user-controller.js (hunk 4 above)
Comment (noisyBrain) [reply controllers/user-controller.js]: Quitado 👌🏻

**Review by soyarielruiz — APPROVED** [review]

**Pull request merged**